.github/workflows/scala.yml

```diff
@@ -15,6 +15,6 @@ jobs:
       - name: Set up JDK
         uses: olafurpg/setup-scala@v13
         with:
-          java-version: adoptium@17
+          java-version: adoptium@11
```

Comment:
Sorry to do this, but maybe we should try to keep 17 and use 8 for releases

Reply:
Ok with releasing for 8, but I think CI tests should include the same JVM we use for release.

I'll look into using a version matrix that includes 8 and 17.